app/build.gradle

```diff
@@ -76,6 +78,10 @@ android {
         lintConfig file("$projectDir/lint.xml")
         warningsAsErrors true
     }
+
+    configurations.all {
```

Comment:
@RamilGabdrakhmanov no. This is example of how to resolve conflict between test and main dependencies